tests/Aspire.Confluent.Kafka.Tests/ConsumerConfigurationTests.cs

```diff
@@ -102,14 +102,111 @@ public void ConnectionNameWinsOverConfigSection(bool useKeyed)
 
         using var host = builder.Build();
         var connectionFactory = useKeyed
-            ? host.Services.GetRequiredKeyedService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value!, "messaging")
-            : host.Services.GetRequiredService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value!);
+            ? host.Services.GetRequiredKeyedService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value, "messaging")
+            : host.Services.GetRequiredService(ReflectionHelpers.ConsumerConnectionFactoryStringKeyStringValueType.Value);
 
-        ConsumerConfig config = GetConsumerConfig(connectionFactory)!;
+        var config = GetConsumerConfig(connectionFactory)!;
 
         Assert.Equal(CommonHelpers.TestingEndpoint, config.BootstrapServers);
     }
 
+    [Theory]
+    [InlineData(true, true, false, false)]
```

Comment:
I think it would be easier to read if we added comments showing which overload each of these corresponds to.

Reply:
Wouldn't that duplicate the content of the switch expression ?

Reply:
Good point.

Reply:
I think the way to get this more explicit is to drop the parameterization of the UT and explode combinations in dedicated UTs.. but that will duplicate the testing code.